apps/web/components/App.tsx

```diff
@@ -66,7 +77,7 @@ const Component = ({
         });
         if (result.status === 200) {
           const res = await result.json();
-          setInstalledAppCount(res.count);
+          setExistingCredentials(res.credentials);
```

Comment:
We now return credentials, not just the count, in order to pass down the credential ID to the DIsconnect button.